.github/workflows/mpet-regression-test.yml

```diff
@@ -15,18 +15,10 @@ jobs:
         shell: bash -l {0}
     steps:
 
-    - uses: actions/checkout@v1
-      with:
-        fetch-depth: 1
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 1
         path: mpet
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 1
-        path: base-mpet
-        ref: ${{ github.base_ref }}
```

Comment:
good catch that we do not use this here, but instead use reference values stored inside the repository